File: reni-backend/src/main/java/pe/gob/mimp/reni/dto/request/ReporteIntervencionXUsuarioListarRequest.java
package pe.gob.mimp.reni.dto.request;

import java.io.Serializable;
import java.util.List;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class ReporteIntervencionXUsuarioListarRequest implements Serializable {
	private static final long serialVersionUID = 1L;

	private String sqlFields;
	private String sqlTables;
	private List<ReporteIntervencionXUsuarioListarDetRequest> listaMapeo;
	private String tipDocUsu;
	private String nroDocUsu;
	private String codUsu;

}

File: reni-frontend/src/main/reni-frontend/src/app/modules/intranet/dto/request/reporte-intervencion-x-usuario-listar.request.ts
import { ReporteIntervencionXUsuarioListarDetRequest } from "./reporte-intervencion-x-usuario-listar-det.request";

export class ReporteIntervencionXUsuarioListarRequest {
    sqlFields: string;
    sqlTables: string;
    listaMapeo: ReporteIntervencionXUsuarioListarDetRequest[];
    tipDocUsu: string;
    nroDocUsu: string;
    codUsu: string;
}